app/views/templates/_form.html.erb

```diff
@@ -84,6 +81,74 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <h1><%= _('Template syntax') %></h1>
+        <%= _('Provisioning templates are generated using the Embedded Ruby (ERB) language.') %>
+        <%= _('The following table and example snippet shows global methods, variables and class methods and how to use them.') %>
+        <%= _('These are guaranteed to work via the safe mode rendering, to ensure a template can do nothing harmful.') %>
+        <% unless Setting[:safemode_render] -%>
+          <%= _('To enable safe mode, navigate to Settings page and enable Safemode rendering option.') %>
+        <% end -%>
+        <%= _('For more info %s.').html_safe % link_to(_('visit our documentation'), documentation_url(controller.documentation_archor)) %>
+        <h2><%= _('Examples') %></h2>
+        <div class="panel-group" id="accordion-markup">
+          <%= render "help_accordion", :title => 'Comments', :contents => (capture { %>
```

Comment:
Yeah I was thinking - when the rest (inline comments in the examples) are not translated - not to do it. Whatever.